app/api/sessions.py

```diff
@@ -59,8 +58,8 @@ def validate_date(self, data, original_data):
     ends_at = fields.DateTime(required=True)
     language = fields.Str()
     slides_url = fields.Url(attribute='slides')
-    videos_url = fields.Url(attribute='videos')
-    audios_url = fields.Url(attribute='audios')
+    videos_url = fields.Url(attribute='video')
```

Comment:
Why is it called videos_url ? It's just video_url ... And also update the db columns accordingly please.